lighthouse-core/lib/i18n/i18n.js

```diff
@@ -21,23 +21,25 @@ const MESSAGE_INSTANCE_ID_REGEX = /(.* \| .*) # (\d+)$/;
 const MESSAGE_INSTANCE_ID_QUICK_REGEX = / # \d+$/;
 
 (() => {
-  // Node usually doesn't come with the locales we want built-in, so load the polyfill if we can.
+  // Node without full-icu doesn't come with the locales we want built-in. Load the polyfill if needed.
+  // See https://nodejs.org/api/intl.html#intl_options_for_building_node_js
+  // The bundler should remove these dependencies, so this will be a no-op in the browser.
```

Comment:
might be worth clarifying "no-op". 

right now it appears to me that we would override a browser that failed to support those 4 locales with `Intl.NumberFormat = undefined`, which would be worse than a no-op. Is it more like `All our environments where IntlPolyfill is empty are guaranteed to support these 4 locales, so this will be a no-op`?

Reply:
Yeah, it shouldn't be possible in Chrome (other browsers never see this code because it's only in the bundle, not in the report) but you never know what Chrome builds are out there and at the least it would be good for the code to make it completely clear it's not possible. I'll fix it.